File: patch_database.py
with open("srcs/server/db/database.go", "r") as f:
    content = f.read()

content = content.replace("sqlStr = strings.ReplaceAll(sqlStr, \"NOW()\", \"CURRENT_TIMESTAMP\")", "sqlStr = strings.ReplaceAll(sqlStr, \"NOW()\", \"CURRENT_TIMESTAMP\")\n\t\t\tsqlStr = strings.ReplaceAll(sqlStr, \"VARCHAR(50)\", \"TEXT\")")

with open("srcs/server/db/database.go", "w") as f:
    f.write(content)

File: patch_database.go
package main

import (
	"fmt"
	"io/ioutil"
	"strings"
)

func main() {
	filePath := "srcs/server/db/database.go"
	contentBytes, err := ioutil.ReadFile(filePath)
	if err != nil {
		fmt.Printf("Error reading file: %v\n", err)
		return
	}
	content := string(contentBytes)

	if strings.Contains(content, "return &DB{Provider: NewPgProvider(pool)}, nil") {
		content = strings.Replace(content, "return &DB{Provider: NewPgProvider(pool)}, nil", "return &DB{Provider: NewPgProvider(pool, nil)}, nil", 1)
		err = ioutil.WriteFile(filePath, []byte(content), 0644)
		if err != nil {
			fmt.Printf("Error writing file: %v\n", err)
			return
		}
		fmt.Println("Successfully modified database.go")
	} else {
		fmt.Println("database.go already patched")
	}
}
